index 55dc338f6..6c0bdcd3f 100644
--- a/workdir/srcs/case_0/include/linux/kcov.h
+++ b/workdir/srcs/case_0/include/linux/kcov.h
@@ -41,6 +41,7 @@ do {						\
 /* See Documentation/dev-tools/kcov.rst for usage details. */
 void kcov_remote_start(u64 handle);
 void kcov_remote_stop(void);
+void notrace kcov_mark_block(u32 i);
 u64 kcov_common_handle(void);
 
 static inline void kcov_remote_start_common(u64 id)
@@ -88,6 +89,7 @@ static inline void kcov_remote_start_common(u64 id) {}
 static inline void kcov_remote_start_usb(u64 id) {}
 static inline void kcov_remote_start_usb_softirq(u64 id) {}
 static inline void kcov_remote_stop_softirq(void) {}
+static inline void kcov_mark_block(u32 i) {}
 
 #endif /* CONFIG_KCOV */
 #endif /* _LINUX_KCOV_H */
