index 451bd8bfa..89c3e7a35 100644
--- a/workdir/srcs/case_0/net/packet/af_packet.c
+++ b/workdir/srcs/case_0/net/packet/af_packet.c
@@ -48,6 +48,7 @@
 
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
+#include <linux/kcov.h>
 #include <linux/ethtool.h>
 #include <linux/filter.h>
 #include <linux/types.h>
@@ -4520,8 +4521,10 @@ static int packet_set_ring(struct sock *sk, union tpacket_req_u *req_u,
 	}
 
 out_free_pg_vec:
+	kcov_mark_block(2);
+	bitmap_free(rx_owner_map);
 	if (pg_vec) {
-		bitmap_free(rx_owner_map);
+		kcov_mark_block(3);
 		free_pg_vec(pg_vec, order, req->tp_block_nr);
 	}
 out:
